src/app/course/goal-create/goal-creation-page.component.scss

```diff
@@ -0,0 +1,63 @@
+tui-input-date{
+    width: 400px;
+}
+tui-select{
+    width: 200px;
+}
+
+.tui-input-number{
+    width: 200px;
+}
```

Comment:
Instead of having these fixed values, you should use TaigaUI form system.
https://taiga-ui.dev/form
See how it set the sizes. Another reason for using this is that the website should look good in the mobile view.
This probably won't look good on mobile. After the fixes in the code, we have to see how this page looks in different screen sizes. Please consider this as you are making these changes so in the next reviews you don't have to change much.